contracts/handler/extensible/SignatureVerifierMuxer.sol

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
```

Comment:
can we disable the rule that requires only a single contract per file here?

Reply:
Done 👍🏾